lib/assessment.rb

```diff
@@ -13,28 +14,29 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.odd?
```

Comment:
`odd?` is one of the tools you'll need, but there's a second. What enumerable method could be used to determine if at least some elements in a collection pass some particular test?